test/modules/logicModule/LM_PC_FundingPot_v1.t.sol

```diff
@@ -95,16 +90,1197 @@ contract LM_PC_FundingPot_v1_Test is ModuleTest {
+            ├── autoClosure should be updated to the new value
+            └── globalAccumulativeCaps should be updated to the new value
+    */
+
+    function testEditRound() public {
+        testCreateRound();
+        uint64 lastRoundId = fundingPot.getRoundCount();
+
+        (
+            uint roundStart_,
+            uint roundEnd_,
+            uint roundCap_,
+            address hookContract_,
+            bytes memory hookFunction_,
+            bool autoClosure_,
+            bool globalAccumulativeCaps_
+        ) = _helper_createEditedRoundParams();
+
+        _helper_callEditRound(
+            lastRoundId,
+            roundStart_,
+            roundEnd_,
+            roundCap_,
+            hookContract_,
+            hookFunction_,
+            autoClosure_,
+            globalAccumulativeCaps_
+        );
+
+        (
+            uint roundStart,
+            uint roundEnd,
+            uint roundCap,
+            address hookContract,
+            bytes memory hookFunction,
+            bool autoClosure,
+            bool globalAccumulativeCaps
+        ) = fundingPot.getRoundGenericParameters(lastRoundId);
+
+        assertEq(roundStart, roundStart_);
+        assertEq(roundEnd, roundEnd_);
+        assertEq(roundCap, roundCap_);
+        assertEq(hookContract, hookContract_);
+        assertEq(hookFunction, hookFunction_);
+        assertEq(autoClosure, autoClosure_);
+        assertEq(globalAccumulativeCaps, globalAccumulativeCaps_);
+    }
+
+    /* Test setAccessCriteria()
+    ├── Given user does not have FUNDING_POT_ADMIN_ROLE
+    │   └── When user attempts to set access criteria
+    │       └── Then it should revert
+    │
+    └── Given user has FUNDING_POT_ADMIN_ROLE
+        ├── Given round does not exist
+        │   └── When user attempts to set access criteria
+        │       └── Then it should revert
+        │
+        ├── Given round is active
+        │   └── When user attempts to set access criteria
+        │       └── Then it should revert
+        │
+        ├── Given AccessCriteriaId is NFT and nftContract is 0x0
+        │   └── When user attempts to set access criteria
+        │       └── Then it should revert
+        │
+        ├── Given AccessCriteriaId is MERKLE and merkleRoot is 0x0
+        │   └── When user attempts to set access criteria
+        │       └── Then it should revert
+        │
+        ├── Given AccessCriteriaId is LIST and allowedAddresses is empty
+        │   └── When user attempts to set access criteria
+        │       └── Then it should revert
+        │
+        └── Given all the valid parameters are provided
+            └── When user attempts to set access criteria
+                └── Then it should not revert
+    */
+
+    function testFuzzSetAccessCriteria_revertsGivenUserDoesNotHaveFundingPotAdminRole(
+        uint8 accessCriteriaEnum_,
+        address user_
+    ) public {
+        vm.assume(accessCriteriaEnum_ >= 0 && accessCriteriaEnum_ <= 3);
```

Comment:
I don't understand why this uses a different vm.assume than the other fuzz tests. Could you explain your intentions?

Reply:
You're right these are unecessary. Removing these, nice catch.

Reply:
Don't think this made sense, I changed all of them to be the same and the tests are still passing so I think this was just a mismatch and not intentional.